### PR TITLE
README: add install instruction

### PR DESCRIPTION
**Pull request by yichengq**
Follow the latest way to build binaries.

/cc @yunxing @wangtuanjie

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,10 +37,10 @@ For now, to install these packages:
 ``` bash
 go get github.com/c-fs/vendor
 go get -u github.com/c-fs/Jerasure
+export GO15VENDOREXPERIMENT=1
```
Comment (wangtuanjie) [inline README.md:42]: `GO15VENDOREXPERIMENT`  requried `go version >= 1.5`.  It is better to explicit go version.

Comment (yichengq) [inline README.md:42]: sure.

Comment (yichengq) [inline README.md:42]: added

**Comment by wangtuanjie** [comment]
LGTM

**Pull request merged**